app/_src/kubernetes-ingress-controller/concepts/architecture.md

```diff
@@ -14,7 +14,24 @@ The {{site.kic_product_name}} configures {{site.base_gateway}} using Ingress or
 
 The figure illustrates how {{site.kic_product_name}} works:
 
-![high-level-design](/assets/images/products/kubernetes-ingress-controller/high-level-design.png "High Level Design")
+<!--vale off-->
+{% mermaid %}
+flowchart LR
+    subgraph Kubernetes cluster
+        direction LR
+        A(<img src="/assets/images/icons/third-party/kubernetes-logo.png" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> API server) --> |events| B(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Controller)
+        B --> |configuration| C(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Kong)
+        C --> D(services)
+    end
+
+    E(Request traffic)
+    E --> C
+
+    %% Change the arrow colors
+    linkStyle 0,1 stroke:#d44324,color:#d44324  
+    linkStyle 2,3 stroke:#b6d7a8
```

Comment:
@lena-larionova I used some ChatGPT help along with copying the values you had in the upgrade strategy guide diagrams for these. I see some of your diagrams left the arrowhead as the default blue/grey color, do we want to color the arrowheads the same as the stroke or leave as the default?

Reply:
Unfortunately, that's a limitation of mermaid :( the arrowheads don't change colour with a style setting. They can only be changed by a global setting on the whole site.